src/components/Info/Info.js

```diff
@@ -62,7 +62,7 @@ function Info({
         background: ${background || modeStyles.background};
         border-left: 2px solid ${borderColor || modeStyles.borderColor};
```

Comment:
Following [designs](https://www.figma.com/file/nfuSVzB1nVJffEgcAITGE7/%F0%9F%90%9D-1hive-UI-kit-v1-Light?node-id=142%3A871) 

We should update this to be `border: 1px solid..`